core/src/main/java/com/linecorp/armeria/internal/common/ByteArrayBytes.java

```diff
@@ -48,16 +53,17 @@ final class ByteArrayHttpData implements HttpData {
         }
     }
 
-    private final byte[] array;
-    private final boolean endOfStream;
-
-    ByteArrayHttpData(byte[] array) {
-        this(array, false);
+    public static ByteArrayBytes of(byte[] array) {
+        return new ByteArrayBytes(array);
     }
```

Comment:
How about adding `Bytes.wrap(byte[])` and `Bytes.wrap(ByteBuf)`?

Reply:
After https://github.com/line/armeria/pull/4505#discussion_r1015461084 is addressed, we don't need the static method anymore, so let's add them when we need them. 😄